sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -96,7 +96,7 @@ export interface AccountSASSignatureValues {
 }
 
 // @public
-export class AnonymousCredential extends Credential {
+export class AnonymousCredential extends Credential_2 {
```

Comment:
This seems a bit strange? Was this there before your change?

Reply:
That's a problem with the latest version of the api extractor. Once #8400 is merged, I'll update this PR and these changes should go away. I manually updated the .md files, looks like I missed datalake :)

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/8389/commits/045ba668f5a4be430cc666a5391e025ecb159691